src/main/kotlin/app/extractors/JavaExtractor.kt

```diff
@@ -0,0 +1,57 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app.extractors
+
+import app.model.DiffContent
+import app.model.Stats
+
+class JavaExtractor : ExtractorInterface {
+    val NAME = "Java"
+
+    val KEYWORDS = listOf("abstract", "continue", "for", "new", "switch",
```

Comment:
It's more for testing purposes. It isn't accurate and it was easiest feature to implement for more complex stats than just language stats.